src/rpdk/core/contract/resource_client.py

```diff
@@ -397,16 +397,22 @@ def _make_payload(self, action, current_model, previous_model=None, **kwargs):
         )
 
     def _call(self, payload):
+        request_without_write_only_properties = prune_properties(
+            payload["requestData"]["resourceProperties"], self.write_only_paths
+        )
+
         payload_to_log = {
-            key: payload[key]
-            for key in [
-                "callbackContext",
-                "action",
-                "requestData",
-                "region",
-                "awsAccountId",
-                "bearerToken",
-            ]
+            "callbackContext": payload["callbackContext"],
+            "action": payload["action"],
+            "requestData": {
+                "resourceProperties": request_without_write_only_properties,
+                "previousResourceProperties": payload["requestData"][
+                    "previousResourceProperties"
+                ],
```

Comment:
`previousResourceProperties` should not be logged directly either

Reply:
you are right. Let me prune it